### PR TITLE
Edited error message for 'Change the Font Size of an Element' challenge

### LINKED ISSUE #7546: Unclear error message.
Challenge [Change the Font Size of an Element](http://www.freecodecamp.com/challenges/change-the-font-size-of-an-element#?solution=%3Cstyle%3E%0A%20%20.red-text%20%7B%0A%20%20%20%20color%3A%20red%3B%0A%20%20%7D%0A%20%20%0A%20%20p%20%7B%0A%20%20%20%20font-size%3A%2016px%3B%0A%20%20%7D%0A%3C%2Fstyle%3E%0A%0A%3Ch2%20class%3D%22red-text%22%3ECatPhotoApp%3C%2Fh2%3E%0A%0A%3Cp%3EKitty%20ipsum%20dolor%20sit%20amet%2C%20shed%20everywhere%20shed%20everywhere%20stretching%20attack%20your%20ankles%20chase%20the%20red%20dot%2C%20hairball%20run%20catnip%20eat%20the%20grass%20sniff.%3C%2Fp%3E%0A%0A%3Cp%3EPurr%20jump%20eat%20the%20grass%20rip%20the%20couch%20scratched%20sunbathe%2C%20shed%20everywhere%20rip%20the%20couch%20sleep%20in%20the%20sink%20fluffy%20fur%20catnip%20scratched.%3C%2Fp%3E%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.116 Safari/537.36</code>.

My code:
```html
<style>
  .red-text {
    color: red;
  }
  
  p {
    font-size: 16px;
  }
</style>

<h2 class="red-text">CatPhotoApp</h2>

<p>Kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff.</p>

<p>Purr jump eat the grass rip the couch scratched sunbathe, shed everywhere rip the couch sleep in the sink fluffy fur catnip scratched.</p>

```
When removing class atribute from both p elements, one of the tests is not passing, and the error is:
> Do not add a class attribute to the second p element, without removing it from the first 
> one.

The description is misleading.
I think better say something like:
> Do not add a class attribute to the second p element.
> First p element should have a class attribute and it's content should remain red.

### PR DESCRIPTION
**Pull request by morganzellers**
Improved an error message on challenge from: 

"Do not add a class attribute to the second p, without removing it from the first 
one."

to:

"Do not add a class attribute to the second p. First p should have a class attribute and it's content should remain red."

Untested on local copy

closes #7546

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/html5-and-css.json [hunk 1]
```diff
@@ -596,7 +596,7 @@
         "assert($(\"p\").length > 1, 'message: You need 2 <code>p</code> elements with Kitty Ipsum text.');",
         "assert(code.match(/<\\/p>/g) && code.match(/<\\/p>/g).length === code.match(/<p/g).length, 'message: Make sure each of your <code>p</code> elements has a closing tag.');",
         "assert.isTrue((/Purr\\s+jump\\s+eat/gi).test($(\"p\").text()), 'message: Your <code>p</code> element should contain the first few words of the provided additional <code>kitty ipsum text</code>.');",
-        "assert($(\"p:not([class])\").length === 1, 'message: Do not add a class attribute to the second <code>p</code> element, without removing it from the first one.');",
+        "assert($(\"p:not([class])\").length === 1, 'message: Do not add a class attribute to the second <code>p</code> element. The first <code>p</code> should have a class attribute and it's content should remain red.);",
```
Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: `red.);` has a missing `'`.
Should be `red.');`

Your code gives :

image

Comment (raisedadead) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: Additionally please modify the sentence to something like:
> The first `p` element should still have a class attribute and it's content should remain `red`.

**Comment by raisedadead** [comment]
@morganzellers

You need to test the changes by running the `gulp`, before submitting.
After manually confirming, test cases need to be run as well with `npm test`.

Thanks again!

File: seed/challenges/01-front-end-development-certification/html5-and-css.json (hunk 1 above)
Comment (sludge256) [inline seed/challenges/01-front-end-development-certification/html5-and-css.json]: Remove the apostrophe from `it's`.

**Comment by BerkeleyTrue** [comment]
Closing as stale. Feel free to re-open when ready.

**Pull request closed without merging**